portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Add Navigation Bar
+const navbar = document.getElementById("navigation-bar");
+
+const navbarList = document.createElement("ul");
+navbarList.setAttribute("class", "navbar-list");
+
+let sections = ["About Me", "Projects", "Contact", "Gallery"];
+
+sections.forEach((sectionName) => {
+    const navbarSection = document.createElement("li");
+    const navbarSectionLink = document.createElement("a");
+
+    navbarSectionLink.setAttribute("href", "javascript:get" + sectionName.replace(/\s/g, '') + "()");   // \s is the regex for "whitespace"
```

Comment:
If you would like to add a comment, it's better to:

#1 add to the line above.
#2 explain the idea instead of the details. For instance, you may say:
"The whitespaces in the section name is removed."

Reply:
Done! Thanks!